warehouse/forklift/metadata.py

```diff
@@ -32,8 +33,8 @@
 
 from warehouse.utils import http
 
-SUPPORTED_METADATA_VERSIONS = {"1.0", "1.1", "1.2", "2.0", "2.1", "2.2", "2.3", "2.4"}
-
+SUPPORTED_METADATA_VERSIONS = {"1.0", "1.1", "1.2", "2.1", "2.2", "2.3", "2.4"}
+assert SUPPORTED_METADATA_VERSIONS.issubset(_VALID_METADATA_VERSIONS)
```

Comment:
```suggestion
```

On second thought, I think we're OK here without the assertion. We generally don't want assertions to raise at runtime because this would cause everything importing this file to fail (e.g. all uploads) rather than just cause the upload with a bad version to fail (which will happen anyways).